log/log.go

```diff
@@ -222,11 +235,24 @@ func Debug(ctx context.Context, fields map[string]interface{}, format string, ar
 
 // extractCallerDetails gets information about the file, line and function that
 // called a certain logging method such as Error, Info, Debug, Warn and Panic.
-func extractCallerDetails() (string, int, string, error) {
+func extractCallerDetails() (string, int, string, string, error) {
```

Comment:
No need to create one struct for 4 attributes. But I thought about adding names for readability so I will add them 👍